app/interactions/pulitzer/setup_post_elements.rb

```diff
@@ -0,0 +1,11 @@
+class Pulitzer::SetupPostElements
+
+  def self.call(post)
```

Comment:
@gomayonqui, over time we've figured out that the class method `call` creates problems for several reasons (thread safety, including modules that only offer instance methods, etc.)

So we've started converting our invitedhome interactions to use an instance, with initialize, call, and attr_accessors for params. Here's an example: https://github.com/tommyfriendhusen08/invited_home/blob/master/app/interactions/apply_option.rb
Could you convert this to use an instance? Your controller will have to call Pulitzer::SetupPostElements.new(...).call instead, but it turns out to have some benefits there too. See this example: https://github.com/tommyfriendhusen08/invited_home/blob/master/app/interactions/apply_option.rb
Because the interaction uses an instance, the instance can be used to access specific errors encountered during the interaction.